Youtube-8m/README.md

```diff
@@ -0,0 +1,370 @@
+# YouTube-8M Tensorflow Starter Code
+
+This repo contains starter code for training and evaluating machine learning
+models over the [YouTube-8M](https://research.google.com/youtube8m/) dataset.
+This is the starter code for our
+[3rd Youtube8M Video Understanding Challenge on Kaggle](https://www.kaggle.com/c/youtube8m-2019)
+and part of the International Conference on Computer Vision (ICCV) 2019 selected
+workshop session. The code gives an end-to-end working example for reading the
+dataset, training a TensorFlow model, and evaluating the performance of the
+model.
+
+## Table of Contents
+
+*   [Running on Your Own Machine](#running-on-your-own-machine)
+    *   [Requirements](#requirements)
+    *   [Download Dataset Locally](#download-dataset-locally)
+    *   [Try the starter code](#try-the-starter-code)
+        *   [Train video-level model on frame-level features and inference at
+            segment-level.](#train-video-level-model-on-frame-level-features-and-inference-at-segment-level)
+        *   [Tensorboard](#tensorboard)
+        *   [Using GPUs](#using-gpus)
+*   [Running on Google's Cloud Machine Learning Platform](#running-on-googles-cloud-machine-learning-platform)
+    *   [Requirements](#requirements-1)
+    *   [Accessing Files on Google Cloud](#accessing-files-on-google-cloud)
+    *   [Testing Locally](#testing-locally)
+    *   [Training on the Cloud over Frame-Level Features](#training-on-the-cloud-over-frame-level-features)
+    *   [Evaluation and Inference](#evaluation-and-inference)
+*   [Create Your Own Dataset Files](#create-your-own-dataset-files)
+*   [Training without this Starter Code](#training-without-this-starter-code)
+*   [Export Your Model for MediaPipe Inference](#export-your-model-for-mediapipe-inference)
+*   [More Documents](#more-documents)
+*   [About This Project](#about-this-project)
+
+## Running on Your Own Machine
+
+### Requirements
+
+The starter code requires Tensorflow. If you haven't installed it yet, follow
+the instructions on [tensorflow.org](https://www.tensorflow.org/install/). This
+code has been tested with Tensorflow 1.14. Going forward, we will continue to
+target the latest released version of Tensorflow.
+
+Please verify that you have Python 3.6+ and Tensorflow 1.14 or higher installed
+by running the following commands:
+
+```sh
+python --version
+python -c 'import tensorflow as tf; print(tf.__version__)'
+```
+
+### Download Dataset Locally
+
+Please see our
+[dataset website](https://research.google.com/youtube8m/download.html) for
+up-to-date download instructions.
+
+In this document, we assume you download all the frame-level feature dataset to
+`~/yt8m/2/frame` and segment-level validation/test dataset to `~/yt8m/3/frame`.
```

Comment:
In your project, the structure of the data folder is different from what is stated here. Can you either adjust it to be fitting with the definition or explain what the difference here is, and how to use it in the project?